src/adapters/auth.ts

```diff
@@ -1,25 +1,47 @@
+/* eslint-disable camelcase */
+
 import { AxiosResponse } from 'axios'
 
+import { postWithJsonBodyHeaders } from 'adapters/headers'
 import requestManager from 'lib/requestManager'
 
 class AuthAdapter {
+  static DEFAULT_PAYLOAD = {
```

Comment:
sorry for getting back to this again. `static` makes class level methods. As we don't need this outside the class how about making it a `const`. What do you think?